Gemfile

```diff
@@ -8,8 +8,6 @@ gem "rails", "~> 7.0.0"
 # Use Puma as the app server
 gem "puma", "~> 5.0"
 # Use SCSS for stylesheets
-gem "sass-rails", ">= 6"
-# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
```

Comment:
You have the wrong comment to remove.

`# Use SCSS for stylesheets`

Reply:
I will fix it. I'm so sorry....

Reply:
fix: [9ff8c72](https://github.com/ranguba/ranguba/pull/38/commits/9ff8c72b50b0e34abeba0fa2e715b9d848d0955c) I've just fixed it.